backend/models/testSession.model.ts

```diff
@@ -13,12 +13,12 @@ export interface Result {
    * a list corresponding to the question list with each field indicating
    * the student's answer
    */
-  answers: [number];
+  answers: number[];
```

Comment:
this and the following changes fix a syntax error that I made while writing these models